packages/web-app-files/src/views/spaces/Project.vue

```diff
@@ -0,0 +1,396 @@
+          this.loadFilesListTask.perform(this, sameRoute, to.params.item)
+        }
+      },
+      immediate: true
+    }
+  },
+  async mounted() {
+    await this.loadResourcesTask.perform(this, false, this.$route.params.item || '')
+
+    document.title = `${this.space.name} - ${this.$route.meta.title}`
+    this.$route.params.name = this.space.name
+
+    const loadSpaceEventToken = bus.subscribe('app.files.list.load', (path) => {
+      this.loadResourcesTask.perform(this, this.$route.params.item === path, path)
+    })
+
+    this.$on('beforeDestroy', () => bus.unsubscribe('app.files.list.load', loadSpaceEventToken))
+  },
+  beforeDestroy() {
+    visibilityObserver.disconnect()
+
+    if (this.$refs.markdownContainer) {
+      this.markdownResizeObserver.unobserve(this.$refs.markdownContainer)
+    }
+  },
+  methods: {
+    ...mapActions('Files', ['loadIndicators', 'loadPreview']),
+    ...mapMutations('Files', [
+      'SET_CURRENT_FOLDER',
+      'LOAD_FILES',
+      'CLEAR_CURRENT_FILES_LIST',
+      'REMOVE_FILE',
+      'REMOVE_FILE_FROM_SEARCHED',
+      'SET_FILE_SELECTION',
+      'REMOVE_FILE_SELECTION'
+    ]),
+    rowMounted(resource, component) {
+      if (!this.displayThumbnails) {
+        return
+      }
+
+      const debounced = debounce(({ unobserve }) => {
+        unobserve()
+        this.loadPreview({
+          resource,
+          isPublic: false,
+          dimensions: ImageDimension.Thumbnail,
+          type: ImageType.Thumbnail
+        })
+      }, 250)
+
+      visibilityObserver.observe(component.$el, { onEnter: debounced, onExit: debounced.cancel })
+    },
+    toggleCollapseMarkdown() {
+      this.markdownCollapsed = !this.markdownCollapsed
+      return this.$refs.markdownContainer.classList.toggle(this.markdownContainerCollapsedClass)
+    },
+    toggleImageExpanded() {
+      this.imageExpanded = !this.imageExpanded
+    },
+    onMarkdownResize() {
+      if (!this.$refs.markdownContainer) {
+        return
+      }
+
+      this.$refs.markdownContainer.classList.remove(this.markdownContainerCollapsedClass)
+      const markdownContainerHeight = this.$refs.markdownContainer.offsetHeight
+
+      if (markdownContainerHeight < 150) {
+        return (this.showMarkdownCollapse = false)
+      }
+
+      this.showMarkdownCollapse = true
+
+      if (this.markdownCollapsed) {
+        this.$refs.markdownContainer.classList.add(this.markdownContainerCollapsedClass)
+      }
+    },
+
+    isResourceInSelection(resource) {
+      return this.selected?.includes(resource)
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+.space-overview {
+  &-image {
+    border-radius: 10px;
+    max-height: 250px;
+    object-fit: cover;
+  }
+
+  &-image.expanded {
+    max-height: 100%;
+    max-width: 100%;
+  }
+
+  &-name {
+    display: -webkit-box;
+    -webkit-line-clamp: 1;
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+  }
+
+  .markdown-container * {
+    color: grey !important;
```

Comment:
Could make use of theme colors here?

Reply:
text-muted or input-text-muted or the likes are supposed to be "grey"-ish

Reply:
🤟